controller.go

```diff
@@ -880,7 +876,19 @@ func (c *Controller) updateService(tunnel *inletsv1alpha1.Tunnel, ip string) err
 		copy.Spec.ExternalIPs = append(copy.Spec.ExternalIPs, ip)
 	}
 
-	_, err = c.kubeclientset.CoreV1().Services(tunnel.Namespace).Update(copy)
+	res, err = c.kubeclientset.CoreV1().Services(tunnel.Namespace).Update(copy)
+	if err != nil {
+		return err
+	}
+
+	// Update Status.LoadBalancer.Ingress
+	copy = res.DeepCopy()
+	copy.Status.LoadBalancer.Ingress = make([]corev1.LoadBalancerIngress, len(copy.Spec.ExternalIPs))
+	for i, ip := range copy.Spec.ExternalIPs {
+		copy.Status.LoadBalancer.Ingress[i] = corev1.LoadBalancerIngress{IP: ip}
```

Comment:
I am wondering about the logic. What if there is > 1 ExternalIP, won't this set them all to the inlets IP?

Reply:
I couldn't sort out what condition resulted in multiple values in the `ExternalIPs` array or in the input IP being an empty string, so I decided to just copy `Spec.ExternalIPs` into the `Status.LoadBalancer.Ingress` array to ensure the two are consistent.